### ISSUE TITLE
Need to be able to set a title attribute on ProgressBar

### ISSUE THREAD

**Issue by IronsUK** [react-bootstrap/react-bootstrap#5777]
Progress bars are flagged for accessibility by axe-core because they render a div with role="progressbar" without a title.

This can be observed on the progress bar documentation page: https://react-bootstrap.github.io/components/progress/

axe-core reports:

_Ensures every ARIA progressbar node has an accessible name_

axe also suggests a fix

_Fix at least one (1) of these issues_:
 - aria-label attribute does not exist or is empty
 - aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
 - Element has no title attribute

It looks like there is no way to specify any of these in the props for ProgressBar

**Comment by kyletsang** [comment]
Can you use this?
https://react-bootstrap.github.io/components/progress/#progress-screenreader-label

**Comment by kyletsang** [comment]
Closed due to inactivity

**Comment by jestrickler** [comment]
> Can you use this?
> https://react-bootstrap.github.io/components/progress/#progress-screenreader-label

No, that does not fix the accessibility violation. The accessible name has to appear on the element with role='progressbar'. I'm seeing the error in Storybook on the accessibility tab. It points [here](https://dequeuniversity.com/rules/axe/4.1/aria-progressbar-name?application=axeAPI) for more info

**Comment by jestrickler** [comment]
So, part of the issue is if I add an aria-label 

`<ProgressBar aria-label='label' ...\>` 

it gets applied to the outer wrapper rather than the element with role='progressbar' 

`<div class="mb-2 progress" aria-label="label"><div role="progressbar" ...></div></div>` 

The label with srOnly creates a new span, but is also not applied as an accessible name to the div with role='progressbar'

**Comment by jestrickler** [comment]
Ok, got a workaround!!! If you nest the ProgressBar, it basically generates the same html, but applies the aria-label to the inner div with role='progressbar' which cleans up the accessibility violation!

`<ProgressBar><ProgressBar aria-label={label} ... /></ProgressBar>`

**Comment by wozzo** [comment]
@kyletsang fyi I've found the same issue.
The workaround mentioned above does resolve it for me too, but it's a bit odd

**Comment by wozzo** [comment]
Gah, nope. Spoke too soon. It resolves the accessibility violation, but breaks my progress bar

**Comment by kyletsang** [comment]
> Gah, nope. Spoke too soon. It resolves the accessibility violation, but breaks my progress bar

Don't think the rendering should be an issue because it does run the same code as if you didn't nest the child ProgressBar.  Do you have a sample reproduction?

**Comment by aganoza** [comment]
Lighthouse was warning me the same. https://dequeuniversity.com/rules/axe/4.1/aria-progressbar-name
Fixed by using isChild prop and wraping the ProgressBar with a div className="progress". 

Maybe worth submitting a pull request where title is passed to the inner div with the progress bar role?

```js
<div className="progress h-100 fs-5">
  <ProgressBar
    animated
    ...
    title="Progress bar title for ARIA"
    isChild
  />
</div>
```
